README.md

```diff
@@ -20,6 +20,18 @@ This app, as well as the ready-to-distribute binaries (see [Deploy](#deploy)), i
 1. Install [homebrew](http://brew.sh/)
 2. `brew update`
 3. `brew install wine`
+4. Add a Windows build to your settings.
+
+```json
+{
+  "electron": {
+    "builds": [
+      {"platform": "win32",
```

Comment:
Re: https://github.com/rissem/meteor-electron/pull/32#issuecomment-163740101 (you should comment in the diff here so that it's easier to follow the discussion :) probably good to show both Mac and Windows here then. It says "Add a Windows build" but if I'm reading this right, just specifying Windows will _stop_ building Mac.